packages/app-bridge/src/types/TemplateLegacy.ts

```diff
@@ -0,0 +1,32 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export type TemplateApiLegacy = {
```

Comment:
What is the reason why you keep `TemplateLegacy` there are no usage yet so it is safe to delete :)

Reply:
Actually, this type is in use in `web-app -> application/client/BrandSdk/AppBridge/v3/AppBridgeBlock.ts`
We decided to keep an old template block for now for a compatibility reasons. You will see usages of that refactored legacy type in a following PR in a `web-app` repo.
Later on, when new template block will be finished and the old one will be successfully migrated we can do the cleanup.